src/ResourceManager/EventHub/Commands.EventHub/Utilities/EventHubsClient.cs

```diff
@@ -493,5 +495,13 @@ public static int ReturnmaxCountvalueForSwtich(int? maxcount)
 
             return returnvalue;
         }
+
+        public static ErrorRecord WriteErrorforBadrequest(ErrorResponseException ex)
+        {
+            ErrorResponseContent errorExtract = new ErrorResponseContent();
+            errorExtract = JsonConvert.DeserializeObject<ErrorResponseContent>(ex.Response.Content);
```

Comment:
As in ServiceBus PR, please add null checks for ex.Response, ex.Response.Content, and errorExtract.error.message

Reply:
Resolved.